src/Features/Core/Portable/PdbSourceDocument/PdbFileLocatorService.cs

```diff
@@ -63,7 +63,7 @@ public PdbFileLocatorService([Import(AllowDefault = true)] ISourceLinkService? s
 
                         // TODO: Support windows PDBs: https://github.com/dotnet/roslyn/issues/55834
                         // TODO: Log results from pdbResult.Log: https://github.com/dotnet/roslyn/issues/57352
-                        if (pdbResult is not null && pdbResult.IsPortablePdb)
+                        if (pdbResult is not null)
```

Comment:
Looks like we support windows pdbs now? Stale comment above?

Reply:
No, we don't, there is just a bug is the debugger where it isn't returning IsPortablePdb correctly. I'm following up with them, but for now have removed it. The Metadata provider.ReadPortablePdbStream call below will still fail, and we'll catch and do the right thing.

Any fix for the bug wouldnt get into P2 so that's why this last minute change happened (literally discovered the bug this morning)